config.yaml

```diff
@@ -148,6 +148,7 @@ matrix:
           - network
           - object_skip
           - object_store
+          - ocr
```

Comment:
add - ocr also in section:

  - env: production_eu-de
    monitoring_zone: eu-nl